src/link.html

```diff
@@ -7,6 +7,7 @@
     <link rel="shortcut icon" type="image/png" href="/favicon.ico"/>
     <title>Enter Code | Hubs by Mozilla</title>
     <link href="https://fonts.googleapis.com/css?family=Zilla+Slab:300,300i,400,400i,700" rel="stylesheet">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">
```

Comment:
`touch-action` seems unsupported on safari, and iOS Safari is the specific browser causing issues with double-tap and pinch-to-zoom. i understand `user-scalable=no` is considered harmful for accessibility because it limits the user's ability to zoom text -- however in both pages we are using it the concern is basically not relevant. for the main hub page, zooming breaks the app, and for the link entry page, the keypad is visually designed to be high contrast, readable, and accessible. there is a minor amount of text (validation errors, etc) on the link page that arguably could be harmed by not being able to zoom, but given that this is causing a usability issue that is much worse, it seems like a fair tradeoff.